.rubocop.yml

```diff
@@ -95,6 +92,7 @@ Style/SpecialGlobalVars:
 
 Style/TrivialAccessors:
   Enabled: false
+  ExactNameMatch: true
```

Comment:
Hmmm. The first declaration expected it to be on, and this second one expected turned it off. If it's disabled like this then is there any value in specifying configuration?

Separate point: do we definitely want it to be switched off?

Reply:
Haha - looks like in the initial commit which added rubocop we have both the rule saying "exact name match" and a rule saying "Enabled: false" - looks like it was just copy-pasted from another project: I think we can just delete this rule, since the code seems fine without it.

Reply:
Ok, I'll give it a try.